src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -926,6 +928,8 @@ class Diagram extends Component<Props> {
       editingNodeItem = node?.contents?.[index]
     } else if (formType === 'trigger') {
       editingNodeItem = node?.conditions?.[index]
+    } else if (formType === 'sub-workflow') {
+      editingNodeItem = index ? 'out' : 'in'
```

Comment:
Just to be clear, this uses the position of the item in the node (block) to determine if the user is editing input or output vars, am I correct ?

Reply:
Yes index 0 is input